src/serverSessionManager.ts

```diff
@@ -26,11 +26,15 @@ export class ServerSessionManager<T extends IDebugSessionLike> {
   private readonly portLeaseTracker: IPortLeaseTracker;
   private disposables: IDisposable[] = [];
   private servers = new Map<string, net.Server>();
+  private host = '127.0.0.1';
 
-  constructor(globalContainer: Container, sessionLauncher: ISessionLauncher<T>) {
+  constructor(globalContainer: Container, sessionLauncher: ISessionLauncher<T>, host?: string) {
```

Comment:
You can make the constructor signature be

```suggestion
  constructor(globalContainer: Container, sessionLauncher: ISessionLauncher<T>, private readonly host = '127.0.0.1') {
```

And then remove the property above and conditional below